### PR TITLE
GTF SECURITY.md

### PR DESCRIPTION
**Pull request by GLOBALTRUSTFUND07**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
+reported vulnerability, what to expect if the vulnerability is accepted or
+declined, etc.
```
Comment (nicc938) [inline SECURITY.md:21]:
```suggestion
# Security Policy

## Supported Versions

Use this section to tell people about which versions of your project are
currently being supported with security updates.

| Version | Supported          |
| ------- | ------------------ |
| 5.1.x   | :white_check_mark: |
| 5.0.x   | :x:                |
| 4.0.x   | :white_check_mark: |
| < 4.0   | :x:                |

## Reporting a Vulnerability

Use this section to tell people how to report a vulnerability.

Tell them where to go, how often they can expect to get an update on a
reported vulnerability, what to expect if the vulnerability is accepted or
declined, etc.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
```
Comment (nicc938) [inline SECURITY.md:1]:
```suggestion
# Security Policy
```

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (nicc938) [inline SECURITY.md:2]:
```suggestion

```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+## Supported Versions
```
Comment (nicc938) [inline SECURITY.md:3]:
```suggestion
## Supported Versions
```
```suggestion
## Supported Versions
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (nicc938) [inline SECURITY.md:4]:
```suggestion

```

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+Use this section to tell people about which versions of your project are
```
Comment (nicc938) [inline SECURITY.md:5]:
```suggestion
Use this section to tell people about which versions of your project are
```

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+currently being supported with security updates.
```
Comment (nicc938) [inline SECURITY.md:6]:
```suggestion
currently being supported with security updates.
```
```suggestion
currently being supported with security updates.
```

File: SECURITY.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (nicc938) [inline SECURITY.md:7]:
```suggestion

```

File: SECURITY.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+| Version | Supported          |
```
Comment (nicc938) [inline SECURITY.md:8]:
```suggestion
| Version | Supported          |
```

File: SECURITY.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+| ------- | ------------------ |
```
Comment (nicc938) [inline SECURITY.md:9]:
```suggestion
| ------- | ------------------ |
```

File: SECURITY.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+| 5.1.x   | :white_check_mark: |
```
Comment (nicc938) [inline SECURITY.md:10]:
```suggestion
| 5.1.x   | :white_check_mark: |
```
```suggestion
| 5.1.x   | :white_check_mark: |
```

File: SECURITY.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+| 5.0.x   | :x:                |
```
Comment (nicc938) [inline SECURITY.md:11]:
```suggestion
| 5.0.x   | :x:                |
```

File: SECURITY.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+| 4.0.x   | :white_check_mark: |
```
Comment (nicc938) [inline SECURITY.md:12]:
```suggestion
| 4.0.x   | :white_check_mark: |
```

File: SECURITY.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+| < 4.0   | :x:                |
```
Comment (nicc938) [inline SECURITY.md:13]:
```suggestion
| < 4.0   | :x:                |
```

File: SECURITY.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (nicc938) [inline SECURITY.md:14]:
```suggestion

```

File: SECURITY.md [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+## Reporting a Vulnerability
```
Comment (nicc938) [inline SECURITY.md:15]:
```suggestion
## Reporting a Vulnerability
```

File: SECURITY.md [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (nicc938) [inline SECURITY.md:16]:
```suggestion

```

File: SECURITY.md [hunk 18]
```diff
@@ -0,0 +1,21 @@
...
+Use this section to tell people how to report a vulnerability.
```
Comment (nicc938) [inline SECURITY.md:17]:
```suggestion
Use this section to tell people how to report a vulnerability.
```

File: SECURITY.md [hunk 19]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (nicc938) [inline SECURITY.md:18]:
```suggestion

```

File: SECURITY.md [hunk 20]
```diff
@@ -0,0 +1,21 @@
...
+Tell them where to go, how often they can expect to get an update on a
```
Comment (nicc938) [inline SECURITY.md:19]:
```suggestion
Tell them where to go, how often they can expect to get an update on a
```

File: SECURITY.md [hunk 21]
```diff
@@ -0,0 +1,21 @@
...
+reported vulnerability, what to expect if the vulnerability is accepted or
```
Comment (nicc938) [inline SECURITY.md:20]:
```suggestion
reported vulnerability, what to expect if the vulnerability is accepted or
```

File: SECURITY.md (hunk 1 above)
Comment (nicc938) [inline SECURITY.md:21]:
```suggestion
declined, etc.
```

**Comment by nicc938** [comment]
this is ok  ?  thxxx

**Review by egorvas — COMMENTED** [review]
test

**Pull request closed without merging**